packages/appstore/src/components/app.tsx

```diff
@@ -12,15 +14,17 @@ type TProps = {
 };
 
 const App: React.FC<TProps> = ({ passthrough: { root_store } }) => (
-    <CashierStoreProvider store={root_store}>
-        <CFDStoreProvider store={root_store}>
-            <StoreProvider store={root_store}>
-                <ExchangeRatesProvider>
-                    <AppContent />
-                </ExchangeRatesProvider>
-            </StoreProvider>
-        </CFDStoreProvider>
-    </CashierStoreProvider>
+    <GrowthBookProvider growthbook={Analytics.getInstances()?.ab?.GrowthBook as any}>
+        <CashierStoreProvider store={root_store}>
+            <CFDStoreProvider store={root_store}>
+                <StoreProvider store={root_store}>
+                    <ExchangeRatesProvider>
+                        <AppContent />
+                    </ExchangeRatesProvider>
+                </StoreProvider>
+            </CFDStoreProvider>
+        </CashierStoreProvider>
+    </GrowthBookProvider>
```

Comment:
As we discussed previously, I'm not approving that solution.
Since `growthbook-react` init real-time connection and ping every second, `AppContent` will be fully re-rendered each second. Downgrade of performance will be significant

Reply:
https://deriv-group.slack.com/archives/CU4NXCZGD/p1705564920481629?thread_ts=1705559559.356609&cid=CU4NXCZGD

please check the message here